apps/sms/views/inbox/js/inbox.js

```diff
@@ -163,8 +164,7 @@ var InboxView = {
     var threadId = node.dataset.threadId;
     var draftId = node.dataset.draftId;
 
-    var threadOrDraft = draftId ?
-      Drafts.byDraftId(+draftId) : Threads.get(+threadId);
+    var threadOrDraft = Threads.get(+(draftId || threadId));
```

Comment:
In this patch I just want to return some unified entity for draft, thread and thread with draft, but don't have good idea right now. What are your thoughts?